src/components/shared/TreeView.js

```diff
@@ -80,20 +78,23 @@ function reactStringWithHighlightedSubstrings(
   return highlighted;
 }
 
-type TreeViewRowFixedColumnsProps = {
-  node: Node,
-  nodeId: IndexIntoCallNodeTable,
+type TreeViewRowFixedColumnsProps<NodeIndex: number, DisplayData: Object> = {
```

Comment:
I think we can do better than `Object` with some clever flow typing ($Keys etc) but that will do for now.